models/Category.js

```diff
@@ -19,6 +19,7 @@ Category.init(
   {
     sequelize,
     freezeTableName: true,
+    timestamps: false,
```

Comment:
Are you trying to include timestamps? If so, does it need to be true?

Reply:
No, I'm trying to remove the timestamps (createdAt/updatedAt)